[2025-02-08T19:03:22.679+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:03:22.694+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-02T00:00:00+00:00 [queued]>
[2025-02-08T19:03:22.702+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-02T00:00:00+00:00 [queued]>
[2025-02-08T19:03:22.704+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:03:22.716+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-02-02 00:00:00+00:00
[2025-02-08T19:03:22.725+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2162) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:03:22.726+0000] {standard_task_runner.py:72} INFO - Started process 2179 to run task
[2025-02-08T19:03:22.725+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-02-02T00:00:00+00:00', '--job-id', '393', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmprkwehmvh']
[2025-02-08T19:03:22.727+0000] {standard_task_runner.py:105} INFO - Job 393: Subtask generate_customer_dim_data
[2025-02-08T19:03:22.767+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-02T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:03:22.834+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-02T00:00:00+00:00'
[2025-02-08T19:03:22.835+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:03:22.850+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T19:03:22.851+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:03:22.857+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:03:22.858+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2025-02-02T00:00:00+00:00, execution_date=20250202T000000, start_date=20250208T190322, end_date=20250208T190322
[2025-02-08T19:03:22.900+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:03:22.912+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:03:22.914+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:43:54.447+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:43:54.464+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-02T00:00:00+00:00 [queued]>
[2025-02-08T20:43:54.473+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-02T00:00:00+00:00 [queued]>
[2025-02-08T20:43:54.474+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:43:54.488+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-02-02 00:00:00+00:00
[2025-02-08T20:43:54.497+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=915) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:43:54.498+0000] {standard_task_runner.py:72} INFO - Started process 937 to run task
[2025-02-08T20:43:54.497+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-02-02T00:00:00+00:00', '--job-id', '523', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmpos999_al']
[2025-02-08T20:43:54.500+0000] {standard_task_runner.py:105} INFO - Job 523: Subtask generate_customer_dim_data
[2025-02-08T20:43:54.545+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-02T00:00:00+00:00 [running]> on host 86932746817d
[2025-02-08T20:43:54.625+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-02T00:00:00+00:00'
[2025-02-08T20:43:54.626+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:43:54.647+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T20:43:54.647+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:43:54.656+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:43:54.657+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2025-02-02T00:00:00+00:00, execution_date=20250202T000000, start_date=20250208T204354, end_date=20250208T204354
[2025-02-08T20:43:54.674+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:43:54.688+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:43:54.690+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
